### PR TITLE
GitHub issue reporting page

### PR DESCRIPTION
**Pull request by KBandipo**
**This pull request makes the following changes**
* Fixes code-collabo/put-repo-name-where-the-issue-is-located-here#putTheIssueNumberHere 

**General checklist**
- [ ] File or folder now contains changes as specified in the issue i worked on
- [ ] I have linked the issue I worked on to this pull request submitted by me

**Testing checklist**

Replace this text with the testing checklist from the issue this pull request fixes.

Ping @code-collabo/web-app

### CODE REVIEW

**Comment by KBandipo** [comment]
Pull Request ready for review @collabo-community/review-web-app

**Comment by Akpjunior94** [comment]
Nice work @KBandipo is there any issue ticket created  or a link that highlights the task asked to do? Well from the PR I see you are trying to create an issue page and on the App level. What you have done looks good to me but if you can point me to where instruction about the task is given then it will help me to leave a more detailed review

**Comment by KBandipo** [comment]
The link is not related to any existing GitHub issues, this is the link for the issue https://discord.com/channels/1148391778492354741/1231877483024875520/1232328067679260803 
Going through the description, I think I still have some fixes to do. I'll do it and push @collabo-community/review-web-app

**Comment by Akpjunior94** [comment]
> The link is not related to any existing GitHub issues, this is the link for the issue https://discord.com/channels/1148391778492354741/1231877483024875520/1232328067679260803 Going through the description, I think I still have some fixes to do. I'll do it and push @collabo-community/review-web-app

Nice. Thanks for the update.

**Comment by KBandipo** [comment]
Pull Request ready for review @collabo-community/review-web-app. This is the link to the issue fixed https://discord.com/channels/1148391778492354741/1231877483024875520/1232328067679260803

File: src/modules/app-home/styles/modules/issue.module.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  word-spacing: 30px;
+  padding: 6px 0;
+  letter-spacing: -5px;
+  text-align: center;
+}
+
+.issueBody {
+  background: #ffc0cb;
+  height: 90vh;
+}
```
[the remark this answers is not in the document]
Comment (Ifycode) [reply src/modules/app-home/styles/modules/issue.module.css:18]: Even for header too - different layout is needed.

> We'll leave this hack just for the purpose of seeing some change in color.

**Review by Ifycode — CHANGES_REQUESTED** [review]
1. The text on the page: `Collabo Community App [/app-home/report-issues]` should be
`Collabo Community App [Report Issues]`

#

2. The browser tab - `/app-home/report-issues | Collabo Community` should be ` Report Issues | Collabo Community`

**Comment by KBandipo** [comment]
Pull Request ready for review, this is the link to the issue fix https://discord.com/channels/1148391778492354741/1148907665414684692/1234890226657263666 @collabo-community/review-web-app

**Comment by Ifycode** [comment]
Closing - designs ongoing for this.

**Pull request closed without merging**